Pre_processing_scripts/pre_prossessing_all.R

```diff
@@ -0,0 +1,2 @@
+source("Pre-processing_scripts/00_Sort_BI_Extracts.R")
+source("Pre-processing_scripts/01_Sort_IT_Extracts.R")
```

Comment:
@Jennit07 , Would you prefer to have this single script to automate all, or not? If so, could you please add other pre-processing scripts to this please? Thank you!

Reply:
I think this is a good idea but i think having this as separate is more beneficial as we dont have to run every script each quarter in this folder and in any particular order. I agree we could clear this up/organise this more. Think we should also have some more documentation notes and structure. E.g. `write_anon_files` notes